_includes/head.html

```diff
@@ -6,7 +6,7 @@
   <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
   <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
 
-  <link rel="stylesheet" href="{{ "/css/main.css" | prepend: site.baseurl }}">
+  <link rel="stylesheet" href="{{ "/assets/stylesheets/main.css" | prepend: site.baseurl }}">
```

Comment:
We don't have an images folder ;)

"Clean" is subjective. I find it cleaner having the assets one level deep rather than 2. Also having 3 folders, `assets`, `assets/stylesheets`, and `assets/images` is hardly cleaner than having just 2.

`/css` is Jekylls default folder name. Sticking to the default probably makes more sense than bikeshedding the name of a folder, no?